documentation/RLC-Swagger-quickstart.md

```diff
@@ -0,0 +1,309 @@
+# Create/Update the ci.yaml
+
+Now, if everything looks good to you, you can submit a PR in azure-sdk-for-js repo with all the changes you made above. Before you do that, you need to add/update the ci.yml file. Depends on whether there's already one in your package folder.
+
+If there's no such file then you can add the following template.
+
+``` yaml
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/purview/
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+    exclude:
+      - feature/v4
+  paths:
+    include:
+      - sdk/purview/
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: purview
+    Artifacts:
+      - name: azure-rest-agrifood-farming
+        safeName: azurerestagrifoodfarming
+```
+
+Please change the paths.include value as your own project path, and change the Artifacts name and safeName into yours.  
+
+If there's already a ci.yml file in your project path. then the only thing you need to do is to add the Artifacts name and safeName of yours into that ci.yml.  
+
+Please notice the Artifacts name should align with your package name. Here the package name is `@azure-rest/agrifood-farming` so the relevant Artifacts name is `azure-rest-agrifood-farming`.
+
+# Create API View
+You may also want to create API View when submitting a PR. You can do it easily by uploading a json file to [API View Website](https://apiview.dev/). The json file is under `<you-sdk-folder>/temp`, and its name ends with `api.json`. For example: `sdk/compute/arm-compute/temp/arm-compute.api.json`.
+
+
+# Prepare PR
+
+RLC can only help you generate SDK code, there is something you need to update manually:
+
+## CHANGELOG.md
+
+CHANGELOG can help customers know the change of new version quicky, so you need to update the it according to the change of this new version. It is also necessary to update release date like `1.0.0-beta.1 (2022-11-11)`(rough time is fine and no need to be very accurate).
+
+## Version Number
+
+You shall update the version number according to [semantic versioning rule](https://semver.org/).
+
+## Test recordings
+
+Please ensure that your test recordings are committed together with your code.
+
+## Fix CI for PR
+You may meet the CI failures after submitting the PR, so please refer to [Troubleshoot CI Failure](./Troubleshoot-ci-failure.md) to fix it.
+
+
+# Release
+
+After the PR is merged, it is time to release package. Here is the [Release Checklist](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/8/Release-Checklist?anchor=prepare-release-script) you should know before release.
+
+# How to configure authentication
```

Comment:
Feel like we should move this section under **Create a swagger/README.md file.under ${PROJECT_ROOT} folder** section